calendar-bundle/src/Resources/config/services.yml

```diff
@@ -7,6 +7,13 @@ services:
             calls:
                 - [setFramework, ['@contao.framework']]
 
+    contao_calendar.cron.generate_feeds:
```

Comment:
Why are we not using a service tag like we do here?

https://github.com/contao/contao/blob/5b8c1d7beb54452b9a2451722b2742d1c026dca7/core-bundle/src/Cron/PurgePreviewLinksCron.php#L22